src/openms/source/DATASTRUCTURES/Compomer.cpp

```diff
@@ -330,16 +333,24 @@ namespace OpenMS
     // how to sort Compomers:
     // first by net_charge
     if (c1.net_charge_ < c2.net_charge_)
-      return true;
+      {
```

Comment:
same here... please check also other changes. Will not point out all of them :)

Reply:
Okay :)